help/destinations/catalog/email-marketing/oracle-eloqua-api.md

```diff
@@ -0,0 +1,165 @@
+
+1. In the **[!UICONTROL Mapping]** step, select **[!UICONTROL Add new mapping]**. You will see a new mapping row on the screen.
+1. In the **[!UICONTROL Select source field]** window, choose the **[!UICONTROL Select attributes]** category and select the XDM attribute or choose the **[!UICONTROL Select identity namespace]** and select an identity.
+1. In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** and select an identity or choose **[!UICONTROL Select custom attributes]** category and select an attribute as needed.
+    * Repeat these steps to add the following mappings between your XDM profile schema and your [!DNL Oracle Eloqua] instance:
+        |Source Field|Target Field| Mandatory|
+        |---|---|---|
+        |`xdm: personalEmail.address`|`Attribute: emailAddress`| Yes |
+        |`IdentityMap: Eid`|`Identity: EloquaId`| Yes |
+
+    * An example using these mappings is shown below:
+    ![Platform UI screenshot example with attribute mappings.](../../assets/catalog/email-marketing/oracle-eloqua-api/mappings.png)
+
+When you are finished providing the mappings for your destination connection, select **[!UICONTROL Next]**.
+
+>[!NOTE]
+>
+>The destination automatically suffixes a unique identifier to the selected segment names on each execution when sending the contact field information to [!DNL Oracle Eloqua]. This ensures the contact field names corresponding to your segment names do not overlap.
+
+## Validate data export {#exported-data}
+
+To validate that you have correctly set up the destination, follow the steps below:
+
+1. Select **[!UICONTROL Destinations]** > **[!UICONTROL Browse]** and navigate to the list of destinations.
+1. Next, select the destination and switch to the **[!DNL Activation data]** tab, then select a segment name.
+![Platform UI screenshot example showing Destinations Activation Data.](../../assets/catalog/email-marketing/oracle-eloqua-api/destinations-activation-data.png)
+
+1. Monitor the segment summary and ensure that the count of profiles corresponds to the count within the segment.
+![Platform UI screenshot example showing Segment.](../../assets/catalog/email-marketing/oracle-eloqua-api/segment.png)
+
+1. Log in to the [!DNL Oracle Eloqua] website, then navigate to the [!DNL Contacts Overview] page to check if the profiles from the segment have been added. To see the segment status drill down into a [!DNL Contact Detail] page and check if the contact field having the selected segment name as its prefix has been created.
```

Comment:
UI elements should be in bold, so that they stand out in the text. this includes UI elements in Eloqua, like [!DNL Contacts Overview] and [!DNL Contact Detail].

minor grammar suggestions: 

To see the segment status, drill down into a [!DNL Contact Detail] page and check if the contact field with the selected segment name as its prefix has been created.

Reply:
Thanks